src/scoreboard/scoreboard.js

```diff
@@ -0,0 +1,122 @@
+// Javascript
+import 'bootstrap/dist/js/bootstrap.min';
+
+// Stylesheets
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './scoreboard.css';
+
+
+$(document).ready(function ()
+{
+    let scoreboardUrl = 'scoreboard.html';
+    let scoreboardApiUrl = 'http://scoreboard.test';
+    let roomName = getUrlParam('room');
+    let timeBetweenUpdates = 30; // seconds
+
+    $('[id^="page-"]').hide();
+
+    if (roomName == undefined) {
+        $('#page-home').show();
+    } else {
+        $('#page-room').show();
+
+        getRoomInfo(roomName);
+    }
+    
+    $('#search-room').click(e => {
+        e.preventDefault();
+
+        window.location = scoreboardUrl + '?room=' + $('#search-room-input').val();
+    });
+
+    $('#create-room').click(e => {
+        e.preventDefault();
+        
+        $.ajax({
+            method: 'POST',
+            url: scoreboardApiUrl + '/rooms',
+        }).done(data => {
+            window.location = scoreboardUrl + '?room=' + data.name;
+        });
+    });
+
+    function getRoomInfo(name) {
```

Comment:
Additional functions may be defined outside the `$(document).ready(() => { ... });` block. Here they are bloat.